core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/EncodingFactory.java

```diff
@@ -38,11 +38,7 @@
 import org.apache.carbondata.core.util.CarbonUtil;
 import org.apache.carbondata.format.Encoding;
 
-import static org.apache.carbondata.format.Encoding.ADAPTIVE_DELTA_INTEGRAL;
-import static org.apache.carbondata.format.Encoding.ADAPTIVE_FLOATING;
-import static org.apache.carbondata.format.Encoding.ADAPTIVE_INTEGRAL;
-import static org.apache.carbondata.format.Encoding.DIRECT_COMPRESS;
-import static org.apache.carbondata.format.Encoding.RLE_INTEGRAL;
+import static org.apache.carbondata.format.Encoding.*;
```

Comment:
Do not use wildcard

Reply:
Ok, I have changed it.

Reply:
I also changed similar problem in columnPage